scene/resources/visual_shader.cpp

```diff
@@ -949,9 +949,6 @@ Vector2 VisualShader::get_node_position(Type p_type, int p_id) const {
 Ref<VisualShaderNode> VisualShader::get_node(Type p_type, int p_id) const {
 	ERR_FAIL_INDEX_V(p_type, TYPE_MAX, Ref<VisualShaderNode>());
 	const Graph *g = &graph[p_type];
-	if (!g->nodes.has(p_id)) {
-		return Ref<VisualShaderNode>();
-	}
```

Comment:
The only one of these that is an obvious oversight is this because of the **ERR_FAIL_COND_V** right below it. 
If you _only_ did this it would be a PR ready to merge.

Reply:
Done #93992.